requirements/test.txt

```diff
@@ -6,6 +6,8 @@ twine==3.2
 mypy>=0.920
 flake8>=3.9.2
 pre-commit>=1.0
+pytest-forked
+sklearn
```

Comment:
Why are these needed?

Reply:
We need to run the hpu tests hence need these package. 
cc: @jerome-habana

Reply:
this is a pytest plugin to spawn processes. Is there any restriction in the number of packages  ?

Reply:
Closing this for now. Please reopen again if there is a limitation

Reply:
Initially, we did have a separate file that include only the hpu requirements. We considered Jirka's input to use the existing file i.e test.txt as it contained the packages which are required to run the hpu tests.
cc : @Borda Please, let us know if we have to commit a new file to install hpu related packages.

Reply:
Will be addressed in a follow up PR